.circleci/src/jobs/deliver_browser_stack.yml

```diff
@@ -0,0 +1,23 @@
+docker:
+  - image: circleci/android:api-29-node
+environment:
+  FASTLANE_SKIP_UPDATE_CHECK: true
+shell: /bin/bash -eo pipefail
+steps:
+  - add_ssh_keys: # Needed for credentials repo
+      fingerprints:
+        - 5b:28:97:36:8b:31:08:ae:5d:ad:19:8a:7d:6f:57:8a
```

Comment:
Does it contain the credentials of browserstack as well?

Reply:
No browserstack credentials are in the environment variables.

Reply:
Then do we need this here?

Reply:
No, we don't. Removed it.